src/asana/helpers.py

```diff
@@ -494,6 +494,11 @@ def _task_followers_from_gh_handles(gh_handles: List[str]) -> List[str]:
         )
         is not None
     ]
+    if not len(followers):
```

Comment:
nit: `not followers` or `len(followers) == 0` would be a little easier to read here